### PR TITLE
Adds title

### PR DESCRIPTION
**Pull request by 01jondaniel**
Removing react template and replaces with our own title.

### CODE REVIEW

**Review by NickHadji — APPROVED** [review]

File: src/App.test.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/Hex 2 RGB/i);
```
Comment (missKatiaPunter) [inline src/App.test.js:6]:
```suggestion
  const titleElement = screen.getByText(/Hex 2 RGB/i);
```

File: src/App.test.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import App from './App';
...
   expect(linkElement).toBeInTheDocument();
```
Comment (missKatiaPunter) [inline src/App.test.js:7]: Make sure you replace all linkElement with titleElement and continue giving meaningful names

Comment (missKatiaPunter) [inline src/App.test.js:7]: Make sure you replace all linkElement with titleElement and continue giving meaningful names

File: src/App.test.js [hunk 3]
```diff
@@ -3,6 +3,8 @@ import App from './App';
...
+  expect(linkElement).toHaveClass('App-Title');
```
Comment (missKatiaPunter) [inline src/App.test.js:8]: Well done. You need to decide if you are going to capitalize all parts of a className or just some.

**Review by missKatiaPunter — CHANGES_REQUESTED** [review]

**Review by missKatiaPunter — APPROVED** [review]

**Pull request merged**